packages/vm/test/tester/index.ts

```diff
@@ -45,6 +45,7 @@ import type { bn128 } from '@ethereumjs/evm'
  * --expected-test-amount: number.        If passed, check after tests are ran if at least this amount of tests have passed (inclusive)
  * --verify-test-amount-alltests: number. If passed, get the expected amount from tests and verify afterwards if this is the count of tests (expects tests are ran with default settings)
  * --reps: number.                        If passed, each test case will be run the number of times indicated
+ * --bls: string.                         Use MCL as a bls library if passed (default: Noble) (state tests only)
```

Comment:
Could we also add this for blockchain tests?

Reply:
Following on this, is there a reason we wouldn't want to use MCL by default in the test runner?  I don't know how many BLS related tests there are but anything that slows down the tests seems undesirable to me since these are the main bottleneck in our CI right now, right?

Reply:
Was a bit lazy and I feel that every new code is this messy test runner is a bit of "lost code" 😂, so I decided to skip. But I can very well add, makes sense for practical reasons and shouldn't be so much extra work.

Reply:
@acolytec3 Yes, guess we can do that for the VM as long as we keep running tests for both implementations in the EVM.